templates/ceph/install.html

```diff
@@ -2,278 +2,131 @@
 
 {% block title %}Install Ceph on Ubuntu{% endblock %}
 
-{% block meta_description %}A step-by-step installation guide to Ceph on your software defined infrastructure.
+{% block meta_description %}
+  A step-by-step installation guide to Ceph on your software defined infrastructure.
 {% endblock %}
 
-{% block meta_copydoc %}https://docs.google.com/document/d/1EmOtHjuEMKhVLQA7GwyrKfCzOKfc8UCEyh-p04-PxzA/edit#{% endblock meta_copydoc %}
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1EmOtHjuEMKhVLQA7GwyrKfCzOKfc8UCEyh-p04-PxzA/edit#
+{% endblock meta_copydoc %}
+
+{% from "_macros/vf_hero.jinja" import vf_hero %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
 
 {% block content %}
 
-<section class="p-strip--suru-topped">
-  <div class="row u-equal-height">
-    <div class="col-7">
-      <h1>Install Ceph on Ubuntu</h1>
-      <p class="u-sv2">Ceph is a storage system designed for excellent performance, reliability, and scalability. However, the installation and management of Ceph can be challenging. The Ceph-on- Ubuntu solution takes the administration minutiae out of the equation through the use of Juju charms. With charms, the deployment of a Ceph cluster becomes trivial as does the scaling of the cluster&rsquo;s storage capacity.</p>
-      <p>Looking for help running Ceph?</p>
+  {% call(slot) vf_hero(
+    title_text='Install Ceph on Ubuntu',
```

Comment:
Just a thought, but we should establish a consistent rule on using single or double marks in Jinja templates

Reply:
Agreed, we seem to use double quotes everywhere else so I'll update to that